components/brave_wallet_ui/components/desktop/portfolio-asset-item/index.tsx

```diff
@@ -16,29 +16,27 @@ export interface Props {
   name: string
   symbol: string
   icon: string
-  assetBalance: string
+  assetBalance: number
   fiatBalance: string
 }
 
-export default class PortfolioAssetItem extends React.PureComponent<Props> {
-
-  navTo = () => () => {
-    this.props.action()
-  }
-
-  render () {
-    const { name, assetBalance, fiatBalance, icon, symbol } = this.props
-    return (
-      <StyledWrapper onClick={this.navTo()}>
-        <NameAndIcon>
-          <AssetIcon icon={icon} />
-          <AssetName>{name}</AssetName>
-        </NameAndIcon>
-        <BalanceColumn>
-          <FiatBalanceText>${fiatBalance}</FiatBalanceText>
-          <AssetBalanceText>{assetBalance} {symbol}</AssetBalanceText>
-        </BalanceColumn>
-      </StyledWrapper>
-    )
+const PortfolioAssetItem = (props: Props) => {
+  const { name, assetBalance, fiatBalance, icon, symbol, action } = props
+  const navTo = () => {
+    action()
   }
+  return (
+    <StyledWrapper onClick={navTo}>
```

Comment:
nit: Instead of creating a new function on each render, could just pass `action` directly here?

Reply:
Good catch! This is fixed.